src/reactnativeapp/pb/demo.proto

```diff
@@ -0,0 +1,315 @@
+// Copyright 2020 Google LLC
```

Comment:
We shouldn't check in this file since the original is from /pb/demo.proto and any changes to the original would cause this file to be updated, too.

Reply:
hey @CharlieTLe, this was discussed above with @puckpuck: https://github.com/open-telemetry/opentelemetry-demo/pull/1781#issuecomment-2484600350

Seems like we're moving towards having these proto files committed for the other services as well

Reply:
The `src/reactnativeapp/pb/demo.proto` file shouldn't be checked in since it's only the [Protobuf definition](https://protobuf.dev/overview/#syntax), and we have a copy of it in `pb/demo.proto`.

What should be checked in instead is the [generated proto code](https://protobuf.dev/programming-guides/proto3/#generated). So the file that should be checked in here is `src/reactnativeapp/protos/demo.ts` (which you've done). The `src/reactnativeapp/pb/demo.proto` should be added to the `.gitignore` file so that it doesn't accidentally get checked in.

Here's an [example](https://github.com/open-telemetry/opentelemetry-demo/pull/1794) of some generated proto code that should be checked in; it's for Python and Go, but the same idea still applies.

Reply:
ah sorry I had misunderstood, I think should be good now with: https://github.com/open-telemetry/opentelemetry-demo/pull/1781/commits/dbec3b7c36afa7d12cbb1b82f259ee4c3a9684d0